### ISSUE TITLE
Process palette is shown on project switch 

### ISSUE THREAD

**Issue by astrauka** [morassman/process-palette#34]
Hello,
I am using [atom project manager](https://github.com/danielbrodin/atom-project-manager).
Each time I switch the project, the process palette is displayed at the bottom.
I am switching projects very often and on each switch I need to toggle the process palette.
Could it be off by default or could there be an option to turn if off?

It seems that `projectChange` handler is causing this
https://github.com/morassman/process-palette/blob/master/lib/main.coffee#L110

Maybe it should be changed to
```coffeescript
@showListView() if @state.visible;
```

Have not tested, as did nit dig into how atom plugins are developed.

**Comment by astrauka** [comment]
@morassman

**Comment by morassman** [comment]
Hello @astrauka 

I installed atom project manager. At first I wasn't able to reproduce this, but after taking a look at project manager's code I saw what was happening.

You are right that `projectChange` is causing this. This method, however, is only called when the list of projects in the current Atom window changes. By default project manager opens each project in its own window, in which case `projectChange` isn't called each time one switches between projects. Each window will then retain its own state which means process palette will remain hidden if it was hidden and vice versa.

After learning this I was able to reproduce the behaviour when I configured project manager to use only one window. It then changes the list of projects each time one switches, which in turn causes the `projectChange` method to be called.

I released a small patch to fix this. I simply removed the line you pointed out. This way it will just stay in whichever visible state it is.

Thank you for your feedback and looking into it!
